### DISCUSSION TITLE
help change item color

### DISCUSSION THREAD

**Discussion by dcantere** [kalkih/mini-media-player#694] [Q&A]
Hello guys!!! I have a problem that I can't solve. I would like to change the color of items in my mini media player. I did some tests with some examples, but without success. can anybody help me?
In the image I attached you can see that the items are practically hidden because of the color, how do I insert a specific color. tks
![Sem título](https://user-images.githubusercontent.com/109360617/187304083-9c74c885-1022-4258-9613-1fd51be74282.jpg)

**Comment by ColtonYYZ** [comment]
Try using card_mod.. I was able to do it but I'm not using the integration in the same way you are:
```
    card_mod:
      style: |
        :host {
        --icon-primary-color: rgba(207,22,207,0.8);
        }
```
![Screenshot 2025-01-02 172454](https://github.com/user-attachments/assets/037888d8-09cc-4555-8793-f4413e3dacd4)
